# Etv2_limb_manuscript
## Main Figures

### Figure 3
|  | Figures | Link | 
| --- | --- | --- | 
|  A plot of 5,178 conserved (>0.8) ATAC-seq intervals sorted by the fold change (FC) between HL Etv2-EYFP+ versus Etv2CKO samples | Fig. 3a | [R](https://colab.research.google.com/github/gongx030/Etv2_limb_manuscript/blob/main/Conservation_limb_ATAC_seq_new_data.ipynb) |
| PCA of different cell populations obtained from limb buds | Fig. 3b | [R](https://github.com/gongx030/Etv2_limb_project/blob/main/chromVAR_limb_ATAC_seq_new_data_CH.ipynb) |
| The heatmap of transcription factor (TF) deviations showed the variations of TF associated chromatin accessibility in each sample | Fig. 3c| [R](https://github.com/gongx030/Etv2_limb_project/blob/main/chromVAR_limb_ATAC_seq_new_data_CH.ipynb) |
|ATAC-seq analysis of the ZRS region (chr5: 29,314,591 â€“ 29,316,266) and the Lmbr1 gene promoter region (chr5:29,377,922-29,379,008). Locations of putative ETS and ETV binding sites are indicated at the bottom. | Fig. 3d | [ZRS](https://genome.ucsc.edu/s/ndsouza/ZRS_region) <br> [Lmbr1_Promoter](https://genome.ucsc.edu/s/ndsouza/LMBR1_promoter) |
| Heatmap showing nucleosome free, mono, di-nucleosome region (shown as white) and nucleosome occupied region (shown as blue) within 1kb up- and down-stream of TF binding sites | Fig. 3e | [R](https://github.com/gongx030/Etv2_limb_project/blob/main/FE_Heatmap_Etv2_limb_v1.ipynb) |
| Limb enhancer peaks present in the each group from the Limb Enhsnacer Genie and Limb p300 ChIP-seq data | Fig. 3f | [R](https://github.com/gongx030/Etv2_limb_project/blob/main/FE_Heatmap_Etv2_limb_v1.ipynb) |

### Figure 5
|  | Figures | Colab link | 
| --- | --- | --- | 
| Overexpression of Etv2 resulted in depletion of nucleosomes at the ZRS. HL E9.5 and E10.5 HL EYFP+ tracks show the ATAC-seq signals of E9.5 hindlimb and Etv2-EYFP+ samples from E10.5 HL, respectively. | Fig. 5I | [ZRS](https://genome.ucsc.edu/s/ndsouza/OCC_Lmbr1) <br> [Lmbr1_Promoter](https://genome.ucsc.edu/s/ndsouza/OCC_Lmbr1_Promoter) |
| The aggregated ATAC-seq Vplot of 1,302 Etv2 motif-centric region which ATAC-seq were significantly different (p-value < 0.1) between WT and Etv2 OE samples | Fig. 5j | [R](https://colab.research.google.com/github/gongx030/Etv2_limb_project/blob/main/Etv2_vplots_limb.ipynb) |

## Supplementary Figures
### Supplementary Figure 6
|  | Figures | Colab link | 
| --- | --- | --- | 
| UMAP showing different clusters for the ATAC seq peaks and peaks similar to the ZRS region cluster together | Fig. 6a | [R](https://colab.research.google.com/github/gongx030/Etv2_limb_project/blob/main/Correlation_and_Pathway_analysis_limb_ATAC_seq_new_data.ipynb) |
| Pathway analysis using the cluster with ZRS region showing the top 5 pathways enriched | Fig. 6b | [R](https://colab.research.google.com/github/gongx030/Etv2_limb_project/blob/main/Correlation_and_Pathway_analysis_limb_ATAC_seq_new_data.ipynb) |

## GEO datasets

| Assay Type | GSE | 
| --- | --- |
| ATACseq| [GSE192865](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192865) | 

